client/webserver/site/src/js/wallets.ts

```diff
@@ -346,58 +257,255 @@ export default class WalletsPage extends BasePage {
     form.style.right = '0'
   }
 
-  /*
-   * Show the markets box, which lists the markets available for a selected
-   * asset.
-   */
-  async showMarkets (assetID: number) {
+  async showSuccess (msg: string) {
     const page = this.page
-    const box = page.marketsBox
-    const card = page.marketsCard
-    const rowInfo = this.rowInfos[assetID]
-    await this.hideBox()
-    Doc.empty(card)
-    page.marketsFor.textContent = rowInfo.name
-    page.marketsForLogo.src = Doc.logoPath(app().assets[assetID].symbol)
-    for (const [host, xc] of Object.entries(app().user.exchanges)) {
-      let count = 0
-      if (!xc.markets) continue
-      for (const market of Object.values(xc.markets)) {
-        if (market.baseid === assetID || market.quoteid === assetID) count++
-      }
-      if (count === 0) continue
-      const marketBox = page.marketCard.cloneNode(true) as HTMLElement
-      const tmpl = Doc.parseTemplate(marketBox)
-      tmpl.dexTitle.textContent = host
-      card.appendChild(marketBox)
-      if (!xc.markets) continue
-      for (const market of Object.values(xc.markets)) {
-        // Only show markets where this is the base or quote asset.
-        if (market.baseid !== assetID && market.quoteid !== assetID) continue
-        const mBox = page.oneMarket.cloneNode(true) as HTMLElement
-        const mktTmpl = Doc.parseTemplate(mBox)
-        mktTmpl.baseSymbol.appendChild(Doc.symbolize(market.basesymbol))
-        mktTmpl.quoteSymbol.appendChild(Doc.symbolize(market.quotesymbol))
-        let counterSymbol = market.basesymbol
-        if (market.baseid === assetID) counterSymbol = market.quotesymbol
-        mktTmpl.logo.src = Doc.logoPath(counterSymbol)
-        // Bind the click to a load of the markets page.
-        const pageData = { host: host, base: market.baseid, quote: market.quoteid }
-        bind(mBox, 'click', () => { app().loadPage('markets', pageData) })
-        tmpl.markets.appendChild(mBox)
-      }
-    }
-    this.animation = this.showBox(box)
+    page.successMessage.textContent = msg
+    this.currentForm = page.checkmarkForm
+    this.forms.forEach(form => Doc.hide(form))
+    Doc.show(page.forms, page.checkmarkForm)
+    page.checkmarkForm.style.right = '0'
+    page.checkmark.style.fontSize = '0px'
+
+    const [startR, startG, startB] = State.isDark() ? [223, 226, 225] : [51, 51, 51]
+    const [endR, endG, endB] = [16, 163, 16]
```

Comment:
I'm not one to nitpick frontend, but I would expect you'd pull these values out of a document element with scss dictating these values.
That said, this is pretty darn simple, but screams style rather than function.

Reply:
Getting r,g,b values from css into javascript is not straightforward. I didn't think it was worth the string parsing required.

Reply:
Figured.  Don't waste time.